libs/documentation/src/lib/pages/layout/layout.component.ts

```diff
@@ -74,4 +74,24 @@ export class ResultsLayoutComponent implements AfterViewInit, OnInit {
     }
 
   }
+
+  updateFields(showInactiveFilterValues){

```

Comment:
Could we modify the example to be bit more explicit? Looking at the federalist build, I wasn't entirely sure what the inactive checkbox was doing in the more filters dialog. Looking through the code, I saw that this example was adding a new state into the location's state select options, but that wasn't immediately obvious in the demo. The example also indicated that an inactive filter value was indicated by `*`, and I didn't see any labels with `*` next to it in the more filters dialog. Let me know if maybe I'm missing something!

Reply:
I updated the inactive option being added in to include an * next to the label. How would you suggest making the example more explicit?

Reply:
Here, I think we can add an inactive filters section in the list of demos for filters rather than in the layout. This way we can isolate the feature and put a 1-2 sentence description on what it does. You can probably copy over the demo for filters hide expression under Components -> Filters, keep just the country, and use that as starting point. It does look like you'll be doing further enhancement on this, so I think it's fine to add the demo in as part of that enhancement.